src/schema.ts

```diff
@@ -44,9 +44,10 @@ const MITaskSchema = object({
   clientFilename: string().optional(),
   originalFilename: string().optional(),
   clientModelFilename: string().optional(),
+  modelFilename: string().optional(),
   status: zenum(['SUCCESS', 'DELETED', 'PENDING_UPLOAD', 'PROCESSING']),
   submitted: number().int().positive(),
-  modelUrl: string().url().optional(),
+  modelUrl: string().optional(),
 }).strict();
 export type MITask = zinfer<typeof MITaskSchema>;
```

Comment:
Is all of this in sync with what's in the `types` library?

Reply:
No, I'll update the `types` library since it's not going to work with the `.url()`.

Reply:
The `types` library is the source of truth. You update it, you break the platform.